scripts/akrtma3d/exakrtma3d_obsprep_lghtn.ksh

```diff
@@ -139,7 +139,7 @@ postmsg "$jlogfile" "$msg"
 cpreq  ${DATA}/LightningInGSI.bufr ${DATA}/LightningInGSI_bufr.bufr
 lghtng_bufr="LightningInGSI_bufr.bufr"
 if [ -f ${DATA}/${lghtng_bufr} ] ; then
-  cpreq ${DATA}/${lghtng_bufr} ${COMINobsproc_rtma3d}/rtma.t${cyc}z.${lghtng_bufr}
+  cpreq ${DATA}/${lghtng_bufr} ${COMINobsproc_rtma3d}/${NET}.t${cyc}z.${lghtng_bufr}
```

Comment:
I believe this should be ${RUN} as this is the output file from the lightning code and dependent on the grid.